metadata-ingestion/tests/integration/kafka-connect/kafka_connect_mces_golden.json

```diff
@@ -986,5 +986,19 @@
         "registryVersion": null,
         "properties": null
     }
+},
+{
+    "entityType": "dataFlow",
```

Comment:
Shouldn't there also be a `dataJob` and `dataJobInputOutput` aspects to produce lineage?

Reply:
Yes, they get generated based on Kafka-topics in the connector, and topics get generated by generating collections in the DB. I spent some time trying to configure the connector and DB to generate a collection->topic, but as I'm not a expert on the topic (pun unintended) I wasn't able to. Working with the test setup was also a bit problematic (on a M1 Mac) and I had constant timing-out issues when running the tests, making iteration slow. 

I can ask around, if someone is able to help troubleshoot the connection issue (might be a timing thing?). But I already tested this with our own Kafka-connect deployment and it was able to extract the topics/dataJobs as expected.

Reply:
ill give it one more try.